Psychtoolbox/PsychProbability/Shuffle.m

```diff
@@ -11,18 +11,78 @@
 % xx/xx/92  dhb  Wrote it.
 % 10/25/93  dhb  Return index.
 % 5/25/96   dgp  Made consistent with sort and "for i=Shuffle(1:10)"
-% 6/29/96	  dgp  Edited comments above.
+% 6/29/96	dgp  Edited comments above.
 % 5/18/02   dhb  Modified code to do what comments say, for matrices.
 % 6/2/02    dhb  Fixed bug introduced 5/18.
 
-[null,index] = sort(rand(size(X)));
-[n,m] = size(X);
-Y = zeros(size(X));
-if n == 1 || m == 1
-	Y = X(index);
-else
-	for j = 1:m
-		Y(:,j)  = X(index(:,j),j);
-	end
+% 9/10/15   niki Add bindDim. so that this function can shuffle with the
```

Comment:
Hi, i've fixed up the help text myself. I just meant moving the chunks of text you've written into the proper places, so it actually displays when one types "help Shuffle". Also some formatting improvements. And the [] brackets only belong into the help text to denote optional parameters, not in the function prototype. See this commit for reference.

https://github.com/kleinerm/Psychtoolbox-3/commit/038eb913ca3f3bfa445fdc27e1a63c1ce6301ffc?diff=unified

Thanks for the improvement.
-mario